my_list.rb

```diff
@@ -0,0 +1,33 @@
+class MyList
+  include Enumerable
+
+  def initialize(*list)
+    @list = list
+  end
+
+  def each(&block)
+    @list.each do |n|
+      block.call(n)
+    end
```

Comment:
- [ ] I would recommend writing the code for the condition `if block is not given` for #each method.

Reply:
done